cmake/Modules/Packages/KOKKOS.cmake

```diff
@@ -98,7 +96,6 @@ else()
                           ${LAMMPS_LIB_KOKKOS_BIN_DIR})
   target_include_directories(lammps PRIVATE ${Kokkos_INCLUDE_DIRS})
```

Comment:
@dalg24, doesn't  the`kokkos` target export the above include directories already?

Reply:
It does.  All is needed is  `target_link_libraries(... Kokkos::kokkos)`

Reply:
I won't make changes here now. This has been working for a long time. Never change a winning team. :wink:

If all goes according to plan, we will raise the minimum CMake version required for LAMMPS after the next stable release to 3.16 and that will allow some significant refactor and simplification of CMake scripting. That would be a point to review this kind of change, too.

LAMMPS has to compile and work on a lot of quirky machines and Kokkos support is complex and rather fragile as it is.

Reply:
Won't argue with that :)

Reply:
Sounds good, let's save that for the CMake refactor then.

Reply:
Sounds good, let's save that for the CMake refactor then.